src/mpigadgetio.cxx

```diff
@@ -130,375 +94,7 @@ void MPIDomainExtentGadget(Options &opt){
 
 ///to update the decomposition based on gadget information
 void MPIDomainDecompositionGadget(Options &opt){
-    #define SKIP2 Fgad[i].read((char*)&dummy, sizeof(dummy));
-    Int_t i,j,k,n,m,temp,pc,pc_new, Ntot;
-    int Nsplit,isplit;
-    Int_t nbins1d,nbins3d, ibin[3];
-    Double_t **histo1d;//for initial projection and estimate of domain decomposition
-    //Double_t ***histo3d,**histo2d;//finer scale histo for more accurate decomposition
-    Double_t *histo3d,*histo2d;//finer scale histo for more accurate decomposition
-    Coordinate *histo3dbinval;
-    Coordinate2D *histo2dbinval;
-    int dummy;
-    FLOAT ctemp[3];
-    char   buf[200];
-    char DATA[5];
-    fstream *Fgad;
-    struct gadget_header *header;
-    Double_t a;
-    int b;
     if (ThisTask==0) {
-        /*
```

Comment:
this `if` (and the correspondig closing `}`) are all what's left in this function; do you want to remove them as well?

Reply:
Sure, I can remove this as well.